src/reactviews/pages/QueryResult/table/plugins/cellSelectionModel.plugin.ts

```diff
@@ -34,17 +40,28 @@ export class CellSelectionModel<T extends Slick.SlickData>
     private selector: ICellRangeSelector<T>;
     private ranges: Array<Slick.Range> = [];
     private _handler = new Slick.EventHandler();
+    private webViewState: VscodeWebviewContext<
+        QueryResultWebviewState,
+        QueryResultReducers
+    >;
 
     public onSelectedRangesChanged = new Slick.Event<Array<Slick.Range>>();
 
-    constructor(private options: ICellSelectionModelOptions = defaults) {
+    constructor(
+        private options: ICellSelectionModelOptions = defaults,
+        webViewState: VscodeWebviewContext<
+            QueryResultWebviewState,
+            QueryResultReducers
+        >,
+    ) {
+        this.webViewState = webViewState;
         this.options = mixin(this.options, defaults, false);
         if (this.options.cellRangeSelector) {
             this.selector = this.options.cellRangeSelector;
         } else {
             // this is added by the node requires above
             this.selector = new CellRangeSelector({
-                selectionCss: { border: "2px dashed grey" },
+                selectionCss: { border: "4px dashed grey" },
```

Comment:
I think the issue with the visibility is less the border thickness and more the color.  What about using `theme.colorBrandBackgroundHover` here?

Reply:
ok, will figure out how to get access to the theming here

Reply:
the webview state should have a `theme` property will all the values

Reply:
Fixed color & updated screenshot